### PR TITLE
Updating sample readmes

### PR DESCRIPTION
**Pull request by sethmanheim**
## Description

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
- [x] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
- [x] If applicable, the public code is properly documented.
- [ ] Pull request includes test coverage for the included changes.
- [x] The code builds without any errors.

### CODE REVIEW

**Comment by sethmanheim** [comment]
@jtaubensee Please review.

File: samples/SampleSender/readme.md [hunk 1]
```diff
@@ -1,173 +1,17 @@
...
-                Console.WriteLine($"{numMessagesToSend} messages sent.");
-            }
-        }
-    }
-    ```
-  
-6. Run the program, and ensure that there are no errors thrown.
-  
-Congratulations! You have now sent messages to an Event Hub.
+The first step is to use the [Azure portal](https://portal.azure.com) to create a namespace of type Event Hubs, and obtain the management credentials your application needs to communicate with the Event Hub. To create a namespace and Event Hub, follow the procedure in [this article](event-hubs-create.md).
```
Comment (jtaubensee) [inline samples/SampleSender/readme.md]: event-hubs-create.md should be a full URL

File: samples/SampleSender/readme.md [hunk 2]
```diff
@@ -1,173 +1,17 @@
-﻿# Get started sending messages to Event Hubs in .NET Core
+# Get started sending messages to Event Hubs in .NET Core
 
-## What will be accomplished
+> [!NOTE]
```
Comment (jtaubensee) [inline samples/SampleSender/readme.md]: [!NOTE] is a docs.microsoft.com specific syntax. Maybe try '>'?

**Comment by jtaubensee** [comment]
@sethmanheim - On second thought, lets take this as an opportunity to move these samples to the azure/azure-event-hubs repo.

File: samples/SampleSender/readme.md (hunk 2 above)
Comment (sethmanheim) [inline samples/SampleSender/readme.md]: I thought it was GitHub flavored Markdown, but better not to take chances...

**Comment by sethmanheim** [comment]
Leave a comment

**Pull request closed without merging**